lib/compareImages.js

```diff
@@ -20,6 +20,10 @@ module.exports = function() {
         return done();
     }
 
+    // This feature makes the output nearly worthless and impossible to read.
+    // Set it high enought that we never hit it.
+    resemble.outputOptions({largeImageThreshold:1000000000})
```

Comment:
Different repo / module, but you're right: https://github.com/lksv/node-resemble.js/blob/master/resemble.js#L353 Fixed.